### PR TITLE
[WIP] Fixes #71, #86, and updated some fields as per latest UX changes for …

### LINKED ISSUE #71: Update outdated terminology
Some terms have evolved and need to be updated.

_the user guide and on boarding documentation were based, in part, upon other non-canonical (e.g. the first several revisions of the workshop document) or outdated sources (e.g. the glossary) and, as such, has issues related to the naming of things, the messaging behind certain capabilities, etc. (e. g. OSIO has neither Projects nor Collaboration Spaces, both of which are older, deprecated terms.)_

Use Todd's GSG as reference: https://drive.google.com/file/d/0B84xd3xHpx5cNlRuaC1aM1NsMVk/view

### PR DESCRIPTION
**Pull request by Preeticp**
…chapter 3. Working with Collaboration Spaces section.
This is done from #71 and #86 perspective, but will be updating some content in a couple of modules.

### CODE REVIEW

File: docs/topics/modules/creating_a_new_space.adoc [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 //.. *Scenario Driven Planning* - Your space is based on Scenarios, which are real-world problems from the user's point of view. The Scenarios deliver Value Propositions and are realized based on (user) Experiences.
 
-. Click btn:[Create] to create your new space. The `How would you like to get started?` dialog box displays the following options:
+. Click btn:[Create] to create your new space. The *How would you like to get started?* dialog box displays the following options:
 
-.. btn:[Import existing code]: Import existing code into the new space and set up a continuous integration (CI) pipeline using OpenShift Online and Jenkins.
+* *<<importing_existing_code,Import existing code>>*: Import existing code into the new space and set up a continuous integration (CI) pipeline using OpenShift Online and Jenkins.
 
-.. btn:[Create a new Quickstart project]: Use wizards to automatically set up your development environment and quickly create projects using the desired tooling.
+* *<<creating_a_new_quickstart_project,Create a new Quickstart project>>*: Use wizards to automatically set up your development environment and quickly create projects>> using the desired tooling.
```
Comment (rkratky) [inline docs/topics/modules/creating_a_new_space.adoc]: `...create projects>>` -- I suppose the `>>` shouldn't be there.

File: docs/topics/modules/creating_a_new_space.adoc [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
 
-.. btn:[No thanks, take me to the new space dashboard]: Go to your new space dashboard to begin working in the space.
+* btn:[No thanks, take me to the new space dashboard]: Go to your new space dashboard to begin working in the space.
```
Comment (rkratky) [inline docs/topics/modules/creating_a_new_space.adoc]: Cap. S?

File: docs/topics/modules/deleting_a_space.adoc [hunk 3]
```diff
@@ -1,11 +1,11 @@
-[id="changing_existing_space.adoc"]
-= Deleting an Existing Space
+[id="deleting_existing_space"]
+= Deleting an existing space
```
Comment (rkratky) [inline docs/topics/modules/deleting_a_space.adoc]: Cap. S?

File: docs/topics/modules/glossary.adoc [hunk 4]
```diff
@@ -3,19 +3,19 @@
...
 
 Pipeline:: Pipelines are a continuous delivery system that, at each step, tests and deploys the code to provide feedback to the user. Examples of such steps are unit testing, performance, integration, and deployment. Each step of the pipeline implements different levels of testing and deployment tasks, provides results, and then passes the code on to the next step.
 
 Scenario-Driven Planning:: A software development methodology focused on real-world problems, or Scenarios, described in the language and from the viewpoint of the user. Scenarios deliver particular Value Propositions and are realized through Experiences.
 
-Work Item:: Work Items describe and keep track of work that needs to be done. A user can be assigned to a Work Item. Each Work Item must be attached to a Collaboration Space and an Area (assigned by default). This can be used to model bugs, tasks, features, ideas, and more.
+Work Item:: Work Items describe and keep track of work that needs to be completed. A Work Item can be assigned to the responsible collaborator. Each Work Item must be attached to a Space and an Area (assigned by default). This can be used to model bugs, tasks, features, ideas, and more.
```
Comment (rkratky) [inline docs/topics/modules/glossary.adoc]: "the responsible collaborator" sounds weird. How about "a member of the project" or something like that?

File: docs/topics/modules/glossary.adoc [hunk 5]
```diff
@@ -28,3 +28,5 @@ Alternate Components:: Components recommended by {osio} to replace license or us
 Unknown licenses:: Licenses unknown to {osio}.
 
 License conflicts:: Licenses that conflict at the stack or component level.
+
+Workspace:: Workspaces are fully configured web based development environments suitable for your code and runtime needs. They are runtime environments where you can modify, test, debug or run your code.
```
Comment (rkratky) [inline docs/topics/modules/glossary.adoc]: - `s/web based/web-based/`
- `s/debug or run/debug, or run/`

File: docs/topics/modules/importing_a_codebase.adoc [hunk 6]
```diff
@@ -1,18 +1,19 @@
...
+. Click btn:[Associate Repository to Space] to add the repository to your Space. The menu:Create[Codebases] sub-tab now displays your repository with information about when it was created and the last commit date.
+. Click *Create workspace* for the target repository from the displayed list. A success message appears when the Workspace is successfully created in Eclipse Che.
+. When your Workspace is ready, in the drop-down list, select the new Workspace for your repository, and then click btn:[Open].
++
+A new browser tab loads and displays the Eclipse Che Workspace with your repository codebase.
 +
 [NOTE]
 ====
-Ensure that your browser allows pop-ups from this page. The Eclipse Che context requires a new tab to start.
+In case a new Workspace is not created, check and ensure that your browser allows pop-ups from this page.
```
Comment (rkratky) [inline docs/topics/modules/importing_a_codebase.adoc]: `s/pop ups/pop-up windows/` (IBM Style Guide)

File: .gitignore [hunk 7]
```diff
@@ -2,3 +2,4 @@ build/
 docs/*/master.xml
 docs/*/master.html
 html/
+index.html
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply .gitignore]: Ok will fix that.

**Comment by rkratky** [comment]
@Preeticp, so, I'm reading IBM Style Guide, and it says:

- _As much as possible, use a lowercase style in running text_
- _Use lowercase for names and terms unless the name or term is a proper noun_
- _Do not capitalize the names of features and components unless they are sold separately or are trademarked._

Examples (incorrect):
- The Dispatcher
- The Image Display Monitor
- To illustrate the concept, create a Class diagram.

Examples (correct)
- The dispatcher
- The image display monitor
- To illustrate the concept, create a class diagram.

Also, about glossaries:

_Use lowercase for glossary terms unless a term is a proper noun or requires capitalization for other reasons._

-----

So, we need to be consistent, but you can probably ignore my capitalization suggestions. Conversely, we should probably decapitalize (what a cool (non-)word) many other occurrences of Spacs, Work Items, Iterations, etc.

@mishaone, WDYT?

**Comment by Preeticp** [comment]
I was planning to reply/suggest in the same vein, though my reasoning was being consistent with the RH way of using sentence case for headings. 
If we use the sentence case for headings,  it is incongruous with our usage of Space/Iterations etc in the running text. 
Thanks for pointing out the IBM suggestion, makes sense.
So a +1 from me.

File: docs/topics/modules/creating_a_new_quickstart_project.adoc [hunk 8]
```diff
@@ -1,16 +1,22 @@
...
-. In the `Name` field, type a unique name for your project.
-. Leave the default value for the `Group Id` and `Version` fields
+. In the *Define Project* section:
+.. Select your organization from the *Organization* drop-down list. The drop-down lists the GitHub organizations where you are a member. A new repository for the Quickstart project will be created in the selected organization.
+.. In the *OpenShift Project Name* field, type a unique name for your project.
+.. Retain the default value for the *Group Id* and *Version* fields.
 . Click btn:[Next].
-. Select a `Build pipeline strategy`. Use the default value (`Release, Stage, Approve and Promote`) for most use cases.
-. Select the `Override Jenkins and POM files` option.
+. In the *Configure Pipeline* section, select a build pipeline strategy. The default value *Release, Stage, Approve and Promote* is apt for most use cases.
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply docs/topics/modules/creating_a_new_quickstart_project.adoc]: 'Build pipeline strategy' is correct, as per UX also verified with Dev.

**Comment by mishaone** [comment]
Re: capitalisation, my original intention was that Collaboration Space should be capitalized since it's a proper noun in OSIO (and subsequently distinguishable from the word soace, i.e. leave a space or press the space key, etc.). It seems that were just calling them Spaces now which is still a proper noun for the equivalent of a working project in OSIO so it may make sense to keep the caps still. I'm open to using either as long as we decide somewhere and then apt consistently across the docs afterwards.

**Comment by rkratky** [comment]
Well, sentence case doesn't apply when the words should be capitalized. No matter the heading rules, we're not going to write 'openshift.io', for example.

What we need to decide (and stay consistent on) is whether spaces, iterations, work items, etc. should be capitalized -- anywhere. If yes, they should be capitalized in headings, too. If not, they shouldn't use caps anywhere in the text.

File: docs/topics/modules/configuring_a_codebase.adoc [hunk 9]
```diff
@@ -1,25 +1,25 @@
...
+. The *Codebases* section displays the following options:
 
- * If codebases are already added to the Collaboration Space, the added codebases are listed and a btn:[+] icon displays at the top right of the view to import more codebases.
+ * If you already have codebases in your Space, they are listed and you can use the btn:[+] icon at the upper-right of the section to import more codebases.
 
- * If no codebases are added, two buttons display:
+ * If there are no codebases in your space, you can:
 
-.. btn:[Add Codebase] to manually add a codebase.
+.. use btn:[Add Codebase] to manually add a codebase.
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply docs/topics/modules/configuring_a_codebase.adoc]: This is the content I want to update, hence the WIP.

**Comment by Preeticp** [comment]
I have left the cases as is for now. Once we decide on it, I will update.

**Comment by rkratky** [comment]
@Preeticp, @mishaone, let's do this: we'll pose the question to the wider team for an executive decision. They need to get their act together on this anyway because the UI is inconsistent in its dealing with this issue.

Suggested formulation: "Do we want to standardize on treating OSIO features as proper nouns or not? If we choose proper nouns, then Work Items, Spaces, Iterations, etc. should be always written capitalized. If not, we should write them as any other words, i.e. never capitalize."

If you agree, I'll pose the question in devtools-saas.

**Comment by mishaone** [comment]
I'm happy with whatever. Personally I would have gone with capitalization because those things are names for things in OSIO. However, it's a good point that if all these other places don't do it despite the same reasoning, we shouldn't either.

**Comment by Preeticp** [comment]
Well IBM has its suggestions for GUI: 'Capitalization in interfaces'. What I see is our UX uses the cases inconsistently in similar situations. I have noted a few while working on the Analytics labeling, and intended to raise a collective issue for them. Let me know if you want me to share them with you.

On the Docs side what we could do is, unless we are using it in action instructions Eg., 'Click **Space**' we could use small case for space when we are simply referring to them, Eg. Typically an iteration comprises a mix of work items, ranging from scenarios, features...
From what I see on Jira: https://issues.jboss.org/browse/RHDEVDOCS-438,  UI uses 'Issues/Agile Board', which is Title case (Headline as per IBM) but the docs use small case for 'issues' Eg...a manager sets up and runs a project while individual users add content and work with issues within a sprint.
As with Misha, I am happy to follow whatever decision we make.

**Comment by rkratky** [comment]
>  unless we are using it in action instructions Eg., 'Click Space' we could use small case for space when we are simply referring to them

I agree with this. To share the specific example we discussed with Preeti today:

_"open a new space by clicking on Space in the menu"_ (that's just to illustrate the difference; I hope such sentence won't end up in actual docs ;)

**Comment by rkratky** [comment]
If we disregard the caps issue, when will this be good to be merged?

**Comment by Preeticp** [comment]
Robert currently I am handling #117 and #96, let me know if you want me to take this up on priority.  The first commit was mostly about #71 and #86. I want to focus on streamlining the content, I am trying to get a proper workflow for the Add codebase section, you seem to have commented on it too. So realistically I should be able to get back to working on this next week.

**Comment by rkratky** [comment]
@Preeticp, yes, I want to get this PR out of the way ASAP because it touches many files and blocks me from doing other changes I need to do. If there's still work to do on this, please, create separate issues for the unfinished tasks (and let me know when I can merge this).

**Comment by Preeticp** [comment]
@rkratky, I have updated #86 to include the case change.
I have also created a new issue #121 for the Codebases workflow change.
Apart from these I think the PR is ready to be merged. Let me know if anything further needs to be done.

**Pull request merged**